AppStoreHW/AppStoreHW/ClientViewController.swift

```diff
@@ -10,14 +10,366 @@ import UIKit
 /// Экран с рекомендациями
 final class ClientViewController: UIViewController {
     
+    // MARK: - Enum
+    
+    private enum Constants {
+        static let whatNews = "Вот что нового"
+        static let orderStatus = "Ваш заказ отправлен."
+        static let productQuantity = "1 товар, доставка завтра"
+        static let processedBy = "Обрабатывается"
+        static let sent = "Отправлено"
+        static let delivered = "Доставлено"
+        static let recomended = "Рекомендуется вам"
+        static let news = "Получайте новости о своем заказе в режиме реального времени."
+        static let notification = "Включите уведомления, чтобы получать новости о своем заказе."
+        static let devices = "ваши устройства"
+        static let showAll = "Показать все"
+        static let title = "Для вас"
+        static let ImageSizeForLargeState: CGFloat = 40
```

Comment:
```suggestion
        static let imageSizeForLargeState: CGFloat = 40
```
у нас переменные и константы с маленькой буквы пишутся)
и ниже тоже